pkg/test/resourcefixture/testdata/basic/essentialcontacts/v1alpha1/essentialcontactscontact/essentialcontactscontact-minimal/_generated_object_essentialcontactscontact-minimal.golden.yaml

```diff
@@ -23,7 +23,7 @@ status:
     reason: UpToDate
     status: "True"
     type: Ready
-  externalRef: projects/${projectNumber}/contacts/11
+  externalRef: projects/${projectNumber}/contacts/7
```

Comment:
By convention, this field should be normalized to `projects/${projectNumber}/contacts/${contactId}

Reply:
I think we just add a normalization in test.

Reply:
done.